src/helpers/queryHelper.js

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable import/no-duplicates */
+import { sequelize } from '../database/models';
+import models from '../database/models';
+
+const byDeparture = async (departure, user, userRole) => {
```

Comment:
This is actually service code. A service is used to interact with another external API. case in point database

Reply:
@didasmbalanya These are the helper functions that are called in the service there is a service which combine these helper functions with the complete functionality.